heat/core/trigonometrics.py

```diff
@@ -99,6 +105,32 @@ def arctan(x, out=None):
     return local_op(torch.atan, x, out)
 
 
+def arctan2(x1, x2) -> dndarray.DNDarray:
+    """
+    Element-wise arc tangent of ``x1/x2`` choosing the quadrant correctly.
+    Returns a new :class:``DNDarray`` with the signed angles in radians between vector (``x2``,``x1``) and vector (1,0)
```

Comment:
This is ok, ```DNDarray``` will now be displayed in markdown code-style. If you want to create a cross-reference link to the DNDarray class, you must use ```:class:~heat.core.dndarray.DNDarray``` .... with the ~ in front the whole module path will not be displayed in the docs. However, since the returntype is already a link, I don't think this is necessary here, just a note for the future :)